### PR TITLE
Artifact links comment github workflow

### PR DESCRIPTION
**Pull request by Asizon**
Add new workflow to create comments on pull request including links to created artifacts.
It runs when CI workflow is finished.

### CODE REVIEW

**Comment by Asizon** [comment]
@McGiverGim done requested changes

**Review by McGiverGim — DISMISSED** [review]
Perfect! We will se if we need more changes once merged and being in use...

**Review by haslinghuis — APPROVED** [review]

File: .github/workflows/artifactLinks.yml
Comment (mathiasvr) [inline .github/workflows/artifactLinks.yml:10]: Using Linux is usually faster/cheaper, or does this action require windows?

File: .github/workflows/artifactLinks.yml
Comment (mathiasvr) [inline .github/workflows/artifactLinks.yml:4]: Should be the "PR" workflow if we only want to use with pull requests

File: .github/workflows/artifactLinks.yml
Comment (mathiasvr) [inline .github/workflows/artifactLinks.yml:20]: I think we should only comment on the PR and not issues as well?

File: .github/workflows/artifactLinks.yml
Comment (Asizon) [reply .github/workflows/artifactLinks.yml:10]: I can try, but i think this script is using windows.

File: .github/workflows/artifactLinks.yml
Comment (Asizon) [reply .github/workflows/artifactLinks.yml:4]: I think we should use that also on linked issues, is also really useful.

File: .github/workflows/artifactLinks.yml
Comment (Asizon) [reply .github/workflows/artifactLinks.yml:20]: I think we should use that also on linked issues, is also really useful.

File: .github/workflows/artifactLinks.yml
Comment (Asizon) [reply .github/workflows/artifactLinks.yml:4]: How I can add it to PR/CI workflow?

Comment (mathiasvr) [reply .github/workflows/artifactLinks.yml:4]:
```suggestion
    workflows: ["PR"]
```

File: .github/workflows/artifactLinks.yml
Comment (McGiverGim) [reply .github/workflows/artifactLinks.yml:10]: It seems is simply a Node application, so is probably that it will work. I've found several repos that use it with Ubuntu, like this:
https://github.com/SyGuS-Org/docs/blob/master/.github/workflows/pr-comment.yaml

They seem to have worked, but the logs are missing (I suppose because they are old, more than 4 months)...

**Review by McGiverGim — APPROVED** [review]
Thanks for the changes!!!!

**Comment by Asizon** [comment]
image

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Pull request merged**
